HW7/marian_hrychak/07.1_task_2_Marian.py

```diff
@@ -0,0 +1,46 @@
+"""
+Write a program that calculates the area of a rectangle,
+triangle and circle (write three functions to calculate the area,
+and call them in the main program depending on the user's choice).
+"""
+
+from math import pi
+
+# Three functions that calculate the area of a rectangle,
+# triangle, and circle.
+
+def rectangle(length, width):
+    """This function calculates the area of a rectangle."""
+    area_rectangle = length * width
```

Comment:
If variable use just in one statement (here just in return) we no need to create this variable.
Just return length * width